src/microbe_stage/Patch.cs

```diff
@@ -39,6 +40,32 @@ public Patch(LocalizedString name, int id, Biome biomeTemplate, PatchSnapshot cu
         this.currentSnapshot = currentSnapshot;
     }
 
+    /// <summary>
+    ///   This contains predefined biome types
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     Explicitly stated values are used as patch ids in classic map
+    ///   </para>
+    /// </remarks>
+    public enum BiomeTypes
+    {
+        Coastal = 0,
+        Estuary = 1,
+        Tidepool = 2,
+        Epipelagic = 3,
+        Mesopelagic = 4,
+        Bathypelagic = 5,
+        Abyssopelagic = 6,
+        Seafloor = 7,
+        Cave = 8,
+        IceShelf = 9,
+        Vents = 10,
+    }
+
+    [JsonProperty]
+    public BiomeTypes BiomeType { get; private set; }
```

Comment:
I think this should be moved to be after the Region property to better follow the order of constructor parameters (that has slight performance impact on json deserialization now). And makes more sense to me to have the ID and other identification properties before this.